integration_tests/integration/checkYourAnswers.cy.ts

```diff
@@ -148,6 +164,16 @@ context('Check Your Answers', () => {
 
       CheckYourAnswersPage.incidentDetailsSummary().should('exist')
       CheckYourAnswersPage.offenceDetailsSummary().should('exist')
+      CheckYourAnswersPage.damageSummary().should('exist')
```

Comment:
Perhaps we could use camel case for this variable, and other places we assign to `Page.verifyOnPage`?

`const checkYourAnswersPage: CheckYourAnswers = Page.verifyOnPage(CheckYourAnswers)`

That may help with readability and consistency as we typically use camel case in other places

Reply:
Good spot, no idea why it's not camel case already - slipped through the net!